packages/frontend-shared/src/components/Tooltip.cy.tsx

```diff
@@ -74,4 +75,50 @@ describe('<Tooltip />', () => {
     cy.get('body').realHover()
     cy.get('.v-popper__popper').should('not.exist', { timeout: 250 })
   })
+
+  it('should allow light theme for Tooltip', () => {
+    cy.mount(() => {
+      return (
+        <div class="flex bg-gray-900 m-100px text-white p-40px w-150px gap-160px">
+          {/* @ts-ignore */}
+          <Tooltip v-slots={slotContents(0)} placement="bottom" color="light" />
+        </div>
+      )
+    })
+
+    cy.get('.default-slot0')
+    .realHover()
+
+    cy.get('.v-popper__popper').should('be.visible').and('have.class', 'cypress-v-tooltip-light')
+    cy.percySnapshot()
+
+    cy.get('body').realHover()
+    cy.get('.v-popper__popper').should('not.exist', { timeout: 250 })
+  })
+
+  it('should allow dark theme for interactive Tooltip', () => {
+    cy.mount(() => {
+      return (
+        <div class="flex m-100px p-4 w-150px gap-160px">
+          {/* @ts-ignore */}
```

Comment:
What is this ts-ignore for?

Reply:
No idea it was a copy and paste. Let me see if I can remove it without check-ts being pissed.